src/kern/exec.c

```diff
@@ -53,6 +68,32 @@ int BPF_PROG(test_void_hook, struct linux_binprm *bprm)
 	// Increment the previous value, as a new process was executed on the
 	// current CPU.
 	(*val)++;
+
+	// Get information about the current process
+	int pid = bpf_get_current_pid_tgid() >> 32;
+	int tgid = (bpf_get_current_pid_tgid() << 32) >> 32;
+	int ppid = 1;
```

Comment:
Why is this ppid 1? is it because init is the defacto parent? We should be setting it to say -1 so that if getting ppid fails, we know this in userspace and not just assume that it is parented to init.